assets/ds-moon-preset.js

```diff
@@ -319,6 +319,15 @@ module.exports = {
         "popo": {
           DEFAULT: withOpacityValue("--popo"),
         },
+        "jiren": {
+          DEFAULT: withOpacityValue("--jiren"),
+        },
+        "heles": {
+          DEFAULT: withOpacityValue("--heles"),
+        },
+        "zeno": {
+          DEFAULT: withOpacityValue("--zeno"),
+        },
```

Comment:
Do the `hover` and `primaryHover` under here have use-cases anymore? If so might be better to alias them to the new tokens.

Reply:
we're not going to use them anymore i believe

Reply:
I decided to leave those untouched for the time being, as primary-hover is in use in Button v1 and "hover" parameter as bg-hover in Dropdown, Progress_linear and Single_item_select v1. 
Should I still remove them?

Reply:
no. just mark it as unused anymore and forget please

Reply:
Explanatory comment added.